### PR TITLE
feat: monolog doctgrine handler

### PR DESCRIPTION
**Pull request by theus77**
|Q              |A  |
|---------------|---|
|Bug fix?       |n|
|New feature?   |y|	
|BC breaks?     |n|	
|Deprecations?  |n|	
|Fixed tickets  |n|	

- monolog doctrine handler
- Clear doctrine logs command

### CODE REVIEW

File: src/Entity/Log.php [hunk 1]
```diff
@@ -0,0 +1,206 @@
+<?php
+
```
Comment (Davidmattei) [inline src/Entity/Log.php:2]:
```suggestion
declare(strict_types=1);
```

File: src/Repository/LogRepository.php [hunk 2]
```diff
@@ -0,0 +1,26 @@
+<?php
+
```
Comment (Davidmattei) [inline src/Repository/LogRepository.php:2]:
```suggestion
declare(strict_types=1);
```

File: src/Entity/Log.php [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+namespace EMS\CommonBundle\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+use Ramsey\Uuid\Doctrine\UuidGenerator;
+use Ramsey\Uuid\UuidInterface;
+
+/**
+ * @ORM\Table(name="log")
```
Comment (Davidmattei) [inline src/Entity/Log.php:10]: Not sure but is log not a reserved keywoard in sql context? Maybe rename table name and entity to LogMessage

**Review by Davidmattei — CHANGES_REQUESTED** [review]

File: src/Repository/LogRepository.php (hunk 2 above)
Comment (theus77) [reply src/Repository/LogRepository.php:2]: Fixed

File: src/Entity/Log.php (hunk 3 above)
Comment (theus77) [reply src/Entity/Log.php]: Fixed

**Review by Davidmattei — APPROVED** [review]

**Pull request merged**